src/api/worker/rest/DefaultEntityRestCache.ts

```diff
@@ -710,9 +715,11 @@ export class DefaultEntityRestCache implements EntityRestCache {
 		// No need to try to download something that's not there anymore
 		if (cached != null) {
 			try {
-				// if the password changed we'll be logged out at this point.
-				// not catching the NotAuthorized or NotAuthenticated Error that results of this to delete the user
-				// allows us to still use the offline login with the old user
+				// in case this is an update for the user: if the password changed we'll be logged out at this point
```

Comment:
I am sorry but I am still confused. "doing this", doing what? Where do we catch those errors? What do we do when they are caught? Wouldn't `_handleProcessingError` throw the entity update out if we are not authorized?